### PR TITLE
update-languages=true for nightly

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/364
PR to main: https://github.com/brave/brave-variations/pull/365

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -1253,27 +1301,6 @@
                 "channel": ["NIGHTLY", "BETA", "RELEASE"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
-        },
-        {
-            "name": "BraveTranslateStudy",
```
Comment (atuchin-m) [inline seed/seed.json:1258]: Moved `BraveTranslateStudy` to the same place as in main branch.

**Review by goodov — APPROVED** [review]

**Comment by iefremov** [comment]
@kjozwiak ptal

**Comment by atuchin-m** [comment]
@kjozwiak In fact we've enabled this thing in Nightly by changing the code.

Currently we don't want to push the things to BETA/STABLE until the models are updated.
So we postpone pushing `update-language` for a few weeks.

**Comment by kjozwiak** [comment]
> @kjozwiak In fact we've enabled this thing in Nightly by changing the code.
> 
> Currently we don't want to push the things to BETA/STABLE until the models are updated. So we postpone pushing `update-language` for a few weeks.

@atuchin-m sounds good 👍 I was going to tackle this next as I just finished running through https://github.com/brave/brave-variations/pull/348#issuecomment-1249692242, https://github.com/brave/brave-variations/pull/362#issuecomment-1250523212, https://github.com/brave/brave-variations/pull/362#issuecomment-1250591189 and https://github.com/brave/brave-variations/pull/370. Just let me know when we're ready to push out `update-language`.

**Pull request closed without merging**